File: https:/github.com/rajesh-kayal-dev/Leetcode.git/2058-ConcatenationOfArray/2058-ConcatenationOfArray.js
// Last updated: 6/15/2026, 9:59:07 AM
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {

    let ans = [2 * nums.length];

    for(let i = 0; i < nums.length; i++){
        ans[i] = nums[i];
        ans[i + nums.length] = nums[i];
    }
    return ans;
};

File: https:/github.com/rajesh-kayal-dev/Leetcode.git/2058-ConcatenationOfArray/2058-ConcatenationOfArray.java
// Last updated: 6/15/2026, 9:59:13 AM
class Solution {
    public int[] getConcatenation(int[] nums) {

        int ans[] = new int [ 2 * nums.length];

        for(int i = 0; i < nums.length; i++){
            ans[i] = nums[i];
            ans[i + nums.length] = nums[i];
        }

        return ans;
        
    }
}